lib/Subscriber/Core/MappingSubscriber.php

```diff
@@ -274,11 +268,19 @@ private function hydrateReferenceField(
                 $this->getDefaultValue($fieldMapping)
             );
 
-            if ($referencedNode) {
-                $accessor->set(
+            try {
+                if ($referencedNode) {
+                    $accessor->set(
                     $fieldName,
-                    $this->proxyFactory->createProxyForNode($document, $referencedNode, $options)
+                    $proxyFactory->createProxyForNode($document, $referencedNode, $options)
                 );
+                }
+            } catch (\Exception $e) {
```

Comment:
It comes from the ProxyManager library.

On Thu, Apr 28, 2016 at 12:43:39AM -0700, Daniel Rotter wrote:
>    In [1]lib/Subscriber/Core/MappingSubscriber.php:
> 
>  >                  );
>  > +                }
>  > +            } catch (\Exception $e) {
> 
>    But why not catching the DocumentManagerException again? The only throw I
>    can encounter in the above code is a RuntimeException inheriting from the
>    DocumentManagerException? We already had the same discussion 😕
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu-document-manager/pull/66#discussion_r61385174
>    2. https://github.com/sulu/sulu-document-manager/pull/66/files/05556aabdb564b020fa388baf338b54d39353978#r61385174
>    3. https://github.com/sulu/sulu-document-manager/pull/66/files/05556aabdb564b020fa388baf338b54d39353978#r61385174